sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/ChatAsyncClient.java

```diff
@@ -395,4 +395,11 @@ public void addEventHandler(ChatEventType chatEventType, RealTimeNotificationCal
     public void removeEventHandler(ChatEventType chatEventType, RealTimeNotificationCallback listener) {
         this.signalingClient.off(chatEventType, listener);
     }
+
+    /**
+     * Enable worker manager which renews registra token
+     */
+    public void enableWorkerManager() {
+        this.pushNotificationClient.setEnableWorkManager(true);
+    }
```

Comment:
We support both approaches - with boolean and without - it just depends on use case and readability. With only what I see in front of me, my bigger question is why are we allowing state changes in the client? We typically strive to avoid this and have all state set in the builder.